### PR TITLE
Improve fake appointment positioning

### PR DESCRIPTION
**Pull request by Olexi**

### CODE REVIEW

File: js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js [hunk 1]
```diff
@@ -135,9 +136,7 @@ class TooltipManyAppointmentsBehavior extends TooltipBehaviorBase {
         return this.scheduler.option("dropDownAppointmentTemplate") === "dropDownAppointment";
     }
 
-    _onAppointmentDragStart(itemData, settings, $target) {
-        this.scheduler.hideAppointmentTooltip();
-
+    _onAppointmentDragStart(itemData, settings, e) {
```
Comment (jsdmitry) [inline js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js]: I suggest renaming the "e" parameter to the  "eventArgs".

File: js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js [hunk 2]
```diff
@@ -187,18 +188,24 @@ class TooltipManyAppointmentsBehavior extends TooltipBehaviorBase {
         newCellIndex === oldCellIndex && appointments._clearItem({ itemData: itemData });
     }
 
-    _prepareDragItem($target, $items, settings) {
-        const targetPosition = translator.locate($target);
+    _prepareDragItem($items, settings, e) {
+        const dragContainerOffset = this._getDragContainerOffset();
```
Comment (jsdmitry) [inline js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js:192]: And this the "e" parameter should be renaming.

**Review by jsdmitry — CHANGES_REQUESTED** [review]
I suggest renaming the "e" parameter to the "eventArgs".

**Review by andrewmakarov — APPROVED** [review]

File: js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js (hunk 1 above)
Comment (Olexi) [reply js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js]: done

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**